account_move_batch_validate/wizard/account_move_validate.py

```diff
@@ -3,7 +3,20 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, fields, models
-from odoo.addons.queue_job.job import job
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from odoo.addons.queue_job.job import job, Job
```

Comment:
```suggestion
    from odoo.addons.queue_job.job import job
```

Reply:
> 
flake8 says:

account_move_batch_validate/models/account_move.py:80:23: F821 undefined name 'Job'

You removed the import of Job, but it's still used in the code (line 80 of account_move.py)

@guewen @alexis-via so better mantain this `Job` import?

Reply:
> > 
> 
> flake8 says:
> 
> account_move_batch_validate/models/account_move.py:80:23: F821 undefined name 'Job'
> 
> You removed the import of Job, but it's still used in the code (line 80 of account_move.py)
> 
> @guewen @alexis-via so better mantain this `Job` import?

@sebalix too